pages/common/nmap.md

```diff
@@ -9,3 +9,11 @@
 - discover hosts in the 192.168.0.X area (no port scan)
 
 `nmap -sn {{192.168.0.1/24}}`
+
+- faster scan of a single host (scans for common ports)
+
+`nmap {{192.168.0.1}}`
```

Comment:
This is identical to the first example. Are you sure you didn't forget something?